packages/react/src/primitives/shared/types.ts

```diff
@@ -100,6 +101,23 @@ type ComponentClassNameKey =
   | 'FieldGroupHasInnerStart'
   | 'FieldShowPassword'
   | 'FieldGroupFieldWrapper'
+  | 'FileUploaderDropZone'
+  | 'FileUploaderDropZoneIcon'
+  | 'FileUploaderDropZoneText'
+  | 'FileUploaderDropZoneButton'
+  | 'FileUploaderFile'
+  | 'FileUploaderFileName'
+  | 'FileUploaderFileSize'
+  | 'FileUploaderFileInfo'
+  | 'FileUploaderFileStatus'
+  | 'FileUploaderFileImage'
+  | 'FileUploaderFileMain'
+  | 'FileUploaderLoader'
+  | 'FileUploaderPreviewer'
+  | 'FileUploaderPreviewerText'
+  | 'FileUploaderPreviewerBody'
+  | 'FileUploaderPreviewerFooter'
+  | 'FileUploaderPreviewerFooterActions'
```

Comment:
@dbanksdesign  ?

Reply:
Generally, yes, our connected components don't have their own classnames because they mostly just compose our primitives. However, the FileUploader is a bit of a special case where it is kind of its own primitive as well. To style the FileUploader component purely using CSS would be difficult without these class names. If you look at the theme and CSS for this component it is _mostly_ custom with the exception of the buttons used in the component. @calebpollman is your concern that these are added to a type in the `primitives` directory and not that we just have them? We might need to rethink our structure for add CSS class names in general then or maybe move this type somewhere else. TBH it was a bit annoying adding new class names because I would have to add them in like 4 different files spread across the UI and React packages.